plot_data/core.py

```diff
@@ -882,7 +882,19 @@ def __init__(self, elements: List[Sample] = None, axes: List[str] = None, point_
         self.axes = axes
         self.point_style = point_style
         self.surface_style = surface_style
-        super().__init__(width=width, height=height, type_="scattermatrix", name=name)
+        self.plots = self._build_multiplot()
+        self.initial_view_on = True
+        super().__init__(width=width, height=height, type_="multiplot", name=name)
+
+    def _build_multiplot(self):
+        subplots = []
+        for line in self.elements[0].values:
+            for col in self.elements[0].values:
```

Comment:
Did you try assigning subplots list in a one-liner comprehension list to see if it is elegant ? Could probably be : 

```py
element = self.elements[0] # Name might not be good
return [Histogram(col) for row in element.values for col in element.values
        if row == col else Scatter(row, col)]
```

Now that I think about it, isn't it weird that we loop twice for col and rows on self.elements[0].values ? Is it what is intended ?

Reply:
I did not know it was possible to double loop in one line, thanks !